crates/environ/src/func_environ.rs

```diff
@@ -450,9 +450,9 @@ impl<'module_environment> cranelift_wasm::FuncEnvironment for FuncEnvironment<'m
         // allocated up front and never moved.
         let (offset_guard_size, heap_style, readonly_base) = match self.module.memory_plans[index] {
             MemoryPlan {
-                memory: _,
                 style: MemoryStyle::Dynamic,
                 offset_guard_size,
+                ..
```

Comment:
If someone adds a new field to `MemoryPlan`, the old code here will oblige them to look at this code and figure out if it needs to think about the new field. With `..`, this code will just silently ignore the new field. There are cases where `..` makes sense, when we're reasonably sure we won't care about other fields which may be added in the future, but I don't think that's the case in the code here.